src/app/qgsappbrowserproviders.cpp

```diff
@@ -795,6 +971,43 @@ void QgsBookmarksItemGuiProvider::populateContextMenu( QgsDataItem *item, QMenu
       QgisApp::instance()->showBookmarks( true );
     } );
     menu->addAction( showBookmarksPanel );
+    menu->addSeparator();
+    QAction *importBookmarks = new QAction( tr( "Import Bookmarks…" ), menu );
```

Comment:
+Spatial (for all import, export, and delete items, won't comment on the remaining ones :) )